doc/man/repoq.1.rst

```diff
@@ -0,0 +1,178 @@
+.. vim: ft=rst sw=2 sts=2 et
+
+==========
+ **repoq**
+==========
+
+------------------------------------------------
+Output repository information for given products
+------------------------------------------------
+
+:Authors:
+:Date: Feb 04, 2016
```

Comment:
Should we update the date as well? What does it represent? 
* date that this man page was written?
* date of the release?
* date of last modification?